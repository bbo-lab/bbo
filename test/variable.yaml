variable_1: "{foo}{bar}"
variable_2: ["{foo}", "{batz}", "{bar}"]
