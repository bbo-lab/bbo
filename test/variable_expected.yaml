variable_1: "foobar"
variable_2: ["foo", "batz", "bar"]
